sagemaker-clarify/clarify-explainability-inference-pipelines/credit_risk_explainability_inference_pipelines.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p>Edit the first image to remove the red typo squiggles from <code>SageMaker</code>  and <code>explainability</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
fixed